### PR TITLE
Add Cyberblock map

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main/resources/data/shardthief/games/cyber_block.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (haykam821) [inline src/main/resources/data/shardthief/games/cyber_block.json:1]: This doesn't have consistency with the other game IDs or this game's name or structure ID.

File: src/main/resources/data/shardthief/games/cyber_block.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	"type": "shardthief:shard_thief",
+	"name": "Shard Thief: Cyberblock",
+	"map": {
+		"structure_id": "shardthief:cyberblock",
+		"spawn_y_offset": 4
+	},
+	"players": {
+		"min": 1,
+		"max": 21
```
Comment (haykam821) [inline src/main/resources/data/shardthief/games/cyber_block.json:10]: Why 21 in particular?

**Review by haykam821 — COMMENTED** [review]
This map doesn't fit with the general design that other Shard Thief maps have.

File: src/main/resources/data/shardthief/games/cyber_block.json (hunk 2 above)
Comment (ghost) [reply src/main/resources/data/shardthief/games/cyber_block.json:10]: I did a test with my discord, it felt balanced at 18-20

File: src/main/resources/data/shardthief/games/cyber_block.json (hunk 1 above)
Comment (ghost) [reply src/main/resources/data/shardthief/games/cyber_block.json:1]: Change it then.

**Comment by ghost** [comment]
That's the point of other users making maps.

**Comment by diabolical17** [comment]
Anyways, I've closed this, no point.

**Comment by Genau6502** [comment]
I'm not impressed by this. The whole point of a map is to be different and provide a slightly different play experience. It looks fine to me and, in fact, I never really like the other shard thief maps. I'm fine with code criticism, just denying a map on the grounds that it's not similar enough to existing maps is ridiculous

File: src/main/resources/data/shardthief/games/cyber_block.json (hunk 2 above)
Comment (ghost) [reply src/main/resources/data/shardthief/games/cyber_block.json:10]: But since it's small, it could be a small player count: 15-16

**Comment by Genau6502** [comment]
On the map itself, I will take a look on the build server later when I have time

**Comment by Gegy** [comment]
Given haykam does ultimately have final say over what goes into their games, another option is to alternatively PR this map to https://github.com/NucleoidMC/Game-Configs/

**Pull request closed without merging**